src/components/shared/contentViewer/index.tsx

```
@@ -0,0 +1,48 @@
+}: ContentViewerProps): JSX.Element => {
+  const options: HTMLReactParserOptions = {
+    replace: (domNode: DOMNode) => {
+      if (
+        domNode instanceof Element &&
+        domNode.name === 'code' &&
+        domNode.attribs?.class
+      ) {
+        const language = domNode.attribs.class.replace('language-', '')
+        try {
+          const highlightedCode = hljs.highlight(
+            (domNode.children[0] as any)?.data || '',
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)